tools/wsdlto/test/src/test/java/org/apache/cxf/tools/wsdlto/jaxws/CodeGenOptionTest.java

```diff
@@ -365,10 +368,10 @@ public void testSuppressGeneratedDateNotPresentOption() throws Exception {
         assertTrue(currentDateInGeneratedAnnotationPresent(str));
         str = IOUtils.readStringFromStream(new FileInputStream(new File(types, "SayHi.java")));
         assertFalse(currentDatePresent(str));
-        assertFalse(currentDateInGeneratedAnnotationPresent(str));
+        assertTrue(currentDateInGeneratedAnnotationPresent(str));
         str = IOUtils.readStringFromStream(new FileInputStream(new File(types, "SayHiResponse.java")));
         assertFalse(currentDatePresent(str));
-        assertFalse(currentDateInGeneratedAnnotationPresent(str));
+        assertTrue(currentDateInGeneratedAnnotationPresent(str));
```

Comment:
I assumed something like this. I did not see the config... is that really what we want here? Because the test's name is `testSuppressGeneratedDatePresentOption` …

Reply:
Yeah, I was caught by confusing test names as well, but this is a change for `testSuppressGeneratedDateNotPresentOption`

```
env.put(ToolConstants.CFG_SUPPRESS_GENERATED_DATE, "true");
```

Reply:
@bmarwell I think it is good to merge, all good from your perspective? thank you!

Reply:
Yes, thanks! 👍🏻